Dnd.Ddd.Model/Character/Character.cs

```diff
@@ -1,8 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
-
+using System.Security.Cryptography;
```

Comment:
Is this namespace even used in this class? I think we can remove this

Reply:
changed